src/emailLogin/runMigration.js

```diff
@@ -0,0 +1,88 @@
+const axios = require('axios');
+const fs = require('fs');
+
+const { getDb } = require('../../db/index');
+
+const { PERSONAL_ACCESS_TOKEN, GITHUB_ORG_NAME: ISOMER_GITHUB_ORG_NAME } = process.env;
+
+const ISOMER_USERS = ['isomeradmin', 'rc-davis', 'lamkeewei', 'pallani', 'LoneRifle', 'prestonlimlianjie', 'alexanderleegs', 'lisatjide', 'kwajiehao', 'gweiying', 'seaerchin', 'isomer-demo', 'NatMaeTan', ' jacksonOGP', 'chienlinggg', 'kathleenkhy', 'joshuajunmingt', 'audreytcy', 'yanjunquek', 'chloe-opengovsg', 'shazlithebestie', 'lennardl', 'oliverli', 'taufiq'];
```

Comment:
These were historical users that might possibly still be in agency teams, particularly those who were around when isomer was first conceived - I think it doesn't matter too much either way, but it might be neater if we remove these members explicitly, what are your thoughts?

Reply:
if so, could we file an issue to write a separate script for removing? 

my understanding is that their permissions as-is would be greater than their permissions after the script ran. is my understanding correct?

Reply:
It depends on how the agency team was set up - from past experience, I've seen some repos where the only team that has write access is the agency's team, instead of the more general isomeradmin + agency github teams that we do for repos now - so in these cases, the isomer members are present directly on the agency teams, and permissions will be downgraded accordingly after the script is run

I was thinking that it's not necessary to remove these users from the github teams since all access will be cut off soon - what I wanted to ensure was that they don't accidentally get added to the site member list we maintain in our own internal db

Reply:
i resolve this cos i trust you